### PR TITLE
Add builder script sales pitch

### PR DESCRIPTION
**Pull request by StripesOO7**
I hope this is a sufficient enough sales pitch. 

otherwise i try to make i more detailed without going overboard but the detailed stuff is over in the scripts repo itself so i didnt want to double it up here

### CODE REVIEW

File: doc/PackCreation.md [hunk 1]
```diff
@@ -230,3 +230,47 @@ And what it looks like back in tracker.json to include that tab group of groups.
 ### **Finishing up**
 And now’s the point where you get to put your pieces into action, and see what’s broken. Copy your whole repository folder into poptracker’s Packs folder, open up Poptracker, and see how it looks. You can make changes to the files in the packs folder with Poptracker open and press the Refresh button to load your changes. From here, double-check that logic functions the way you expect it to when items are added / removed, confirm that your maps are readable, and make sure the layouts don’t waste a bunch of space – not everyone has a nice monitor to see tiny pixels.  
 If you created a variant (way back in the Manifest.json), then you’re going to need to create the files that get loaded in place of the default JSON files – but that’s really just more of what you’ve already been doing. Read the documentation in the example pack, and you’ll be good to go.
+
+
+
+# Poptracker Builder-Script
```
Comment (black-sliver) [inline doc/PackCreation.md:236]:
```suggestion
## Poptracker Builder-Script
```

I approve >1 blank line, but more than 2 is never necessary. Also `#` is document title and `##` is highest level heading, so should probably be `##`

File: doc/PackCreation.md [hunk 2]
```diff
@@ -230,3 +230,47 @@ And what it looks like back in tracker.json to include that tab group of groups.
...
+
+You can get the Script [here](https://github.com/StripesOO7/poptracker-pack-builder)
+**The Readme in this repo will take a deeper dive into how to use the script**
+
+**This does not mean you can skip on learning how everything works! It only cuts down the inital time to build those structres.**
+
+The result will be an ready to use base-tracker that can connetect to Archipelago as well as display, and autotrack all items and locations (after providing at least the needed images for items and maps)
+
+
+#### Things you still need to manually do after using the script:
```
Comment (black-sliver) [inline doc/PackCreation.md:261]:
```suggestion
#### Things you still need to manually do after using the script:

```
For style consistency, I'd add a blank line there

File: doc/PackCreation.md [hunk 3]
```diff
@@ -230,3 +230,47 @@ And what it looks like back in tracker.json to include that tab group of groups.
...
+
+
+#### Where do I get the Archipelago Item- and Location-IDs from?
+
+If the game is on the "Supported Games" list, you can look at the [datapackage](https://archipelago.gg/datapackage) on the main website.
+
+If it's an unsupported game, you can either skim through that specific world's code or launch a local copy of the Webhost with the unsupported game in the `custom_worlds` folder and then visit http://localhost/datapackage. If you're unsure how to launch a local Webhost, there is a guide on [running the Archipealgo Webhost locally from source](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/running%20from%20source.md#running-from-source).
+
+
+Any issues with the Builder Script should be reported as issues in the [Script's repo](https://github.com/StripesOO7/poptracker-pack-builder/issues) NOT in the Poptracker repo.
```
Comment (black-sliver) [inline doc/PackCreation.md:276]:
```suggestion
Any issues with the Builder Script should be reported as issues in the [Script's repo](https://github.com/StripesOO7/poptracker-pack-builder/issues) NOT in the PopTracker repo.
```

File: doc/PackCreation.md [hunk 4]
```diff
@@ -230,3 +230,47 @@ And what it looks like back in tracker.json to include that tab group of groups.
...
 If you created a variant (way back in the Manifest.json), then you’re going to need to create the files that get loaded in place of the default JSON files – but that’s really just more of what you’ve already been doing. Read the documentation in the example pack, and you’ll be good to go.
+
+
+## Poptracker Builder-Script
+
+While working with some packs to help them get started and while creating a few packs myself, I noticed some common pain points at the start of the process. My desire to automate the starting steps began to grow and I hoped to mitigate at least some of these pain points. After getting through the first steps of creating your own tracker, you've most likely noticed two major hurdles that need to be overcome before even starting to see anything in Poptracker or having autotracking with Archipelago working:
```
Comment (black-sliver) [inline doc/PackCreation.md:237]:
```suggestion
While working with some packs to help them get started and while creating a few packs myself, I noticed some common pain points at the start of the process. My desire to automate the starting steps began to grow and I hoped to mitigate at least some of these pain points. After getting through the first steps of creating your own tracker, you've most likely noticed two major hurdles that need to be overcome before even starting to see anything in PopTracker or having autotracking with Archipelago working:
```

**Review by black-sliver — COMMENTED** [review]
Other than the two suggestions, one other thing I noticed: you write "auto tracking" in various different ways. I made the same mistake in PACKS.md, but here is the opportunity for you to pick your preferred one :D

**Comment by StripesOO7** [comment]
Do you mean in this sales pitch or inside the repos readme?
Because i only see it written as "autotracking" all one word but once capitalized and twice all lowercase. Which looks to me like i should use it all lowercase all the time then.

**Comment by black-sliver** [comment]
I was referring to the one with an upper case 'A', yeah. I've seen a another different variation, but that was not in your change, sorry for the confusion.

**Review by black-sliver — APPROVED** [review]
Thanks!

**Pull request merged**